dependencies/python/dependencies.bzl

```diff
@@ -0,0 +1,10 @@
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+
+
+def io_bazel_rules_python():
+    git_repository(
+        name = "io_bazel_rules_python",
+        # Grakn python rules
+        remote = "https://github.com/graknlabs/rules_python.git",
+        commit = "4443fa25feac79b0e4c7c63ca84f87a1d6032f49"  # sync-marker: do not remove this comment, this is used for sync-dependencies by @io_bazel_rules_python
```

Comment:
Remove the `# sync-marker: ...` comment, @flyingsilverfin. This dependency is not synced by Grabl, so this comment is not true. cc: @jmsfltchr @lolski